Introduction.adoc

```diff
@@ -1,26 +1,36 @@
+== Introduction
+
 ////
 I'd say this section is a good place to give a very sparse high level idea of
 what the synth is about, provide links to subsections, and provide a quickstart
 ////
 
-ZynAddSubFX is a powerful multi-platform open-source software sound synthesizer.
-Zyn provides a variety of sound creation options including 3 core synthesis
-engines, 10+ filter options, lfos/envelopes for modulation, microtonal support,
-multi-parameter automation support, as well as many other features.
-Zyn can be run both as a standalone application as well as a VST/LV2/DSSI plugin
-within your favorite DAW.
-ZynAddSubFX can be controlled through multiple interfaces, though this manual
-will focus on the new and powerful Zyn-Fusion interface.
+ZynAddSubFX is a multi-platform open-source software sound synthesizer, providing a variety of sound creation options including:
+
+* 3 core synthesis engines
+* 10+ filter options
+* LFOs/envelopes for modulation
+* Microtonal support
+* Multi-parameter automation support
+
+ZynAddSubFX can be run both as a standalone application as well as a VST/LV2/DSSI plugin
+within your favorite DAW. ZynAddSubFX can be controlled through multiple interfaces, though this manual
+will focus on the Zyn-Fusion interface.
+
 
+// Note: celestehorgan – I would actually suggest removng this section entirely. It will end up being hard to maintain in future/will get out of sync, and it doesn't add much.
 === About the User Manual
 
-The Manual was divided into several sections:
+The Manual is divided into the following sections:
+
+Introduction:: An overview of ZynAddSubFX, installation instructions, and quick start tutorials.
+Basic:: Information on the ZynAddSubFX interface, synthesizer modules, synthesis engines, and more.
+Advanced:: Information on performing advanced tasks in ZynAddSubFX.
+Glossary:: A glossary of specific terms and words.
+Routing Diagrams:: A set of signal routing diagrams which display the inner workings of the synthesizer.
```

Comment:
I agree this needs some tweaks, but the changes shift from something with a flow and consistent voice into a collection of basic statements which do concisely convey the information, but in a repetitive manner. For a document which is going to be responsible for teaching about how complex technical matters should impact users and guide them through the system the language does need to be more lively to continue to engage them.